### ISSUE TITLE
[BUG] - AFS not honouring ALT in window

### ISSUE THREAD

**Issue by KJA1582** [invernyx/md11-bugs#1277]
### Describe the bug

When changing the INIT Page altitudes, AFS commands climb thrust and pitches upwards even though AFS ALT window value has not changed

### Steps to reproduce

Establish cruise with INIT Page crz level and AFS ALT window matching
Increase INIT page crz level
observe climb

### Expected behaviour

AFS honouring AFS ALT window

### Evidence

https://youtu.be/2nmX5waF0lg

File too big for GH

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Crash

No

### Phase of flight

Cruise

### Pitch

3°

### Bank

0°

### Heading

287

### Altitude

FL320

### Departure

OMDB

### Arrival

EDDF

### Cruise level

FL320

### ZFW

199848 kgs

### CG

24.1

### Fuel

42950 kgs

### Cost index

80

### Route

[OMDBEDDF_PDF_1716904292.pdf](https://github.com/invernyx/md11-bugs/files/15473415/OMDBEDDF_PDF_1716904292.pdf)

### Location

19nmi before SRT

### Agreement

- [X] Yes

**Comment by turbofandude** [comment]
This will be solved by causing a new cruise altitude > your altitude to revert to CLB mode (it'll also fix the lack of CLB THRUST during S/C)

**Comment by KJA1582** [comment]
Yes, but shouldn't AFS not actually climb if the window is lower then the target?
Unless switching from CRZ to CLB mode fixes this

**Comment by joon44** [comment]
> Yes, but shouldn't AFS not actually climb if the window is lower then the target? Unless switching from CRZ to CLB mode fixes this

ALT window is always the absolute ceiling or floor altitude the plane climbs/descends to.

**Comment by turbofandude** [comment]
Resolved in 1.0.42+
